### DISCUSSION TITLE
How to auto generate AI image after each prompt?

### DISCUSSION THREAD

**Discussion by YW5555** [LostRuins/koboldcpp#748] [Q&A]
I like the new AI image generation feature but is there a way to enable auto generate new image after each prompt? I tried ticking "auto generate" option in settings but it's not doing anything.

I'm using the ROCm fork with AMD 6900XT.

**Comment by LostRuins** [comment]
Auto generate adds a new image every 750 characters (about 100 words). If you enable it and keep generating images will be inserted. If you need more images just click the 'Add IMG' button and select 'Automatic'

**Reply by wh33t** [reply]
I'm still learning about image gen in koboldcpp. I'm curious what prompt gets sent to the image generator. Does it just send the last few sentences or something?

**Reply by LostRuins** [reply]
You can select an image to see what was the prompt that generated it
